apps/root/src/pages/home/frame/index.tsx

```diff
@@ -39,24 +41,50 @@ const HomeFrame = () => {
   const [selectedWalletOption, setSelectedWalletOption] = React.useState<WalletOptionValues>(ALL_WALLETS);
   const dispatch = useAppDispatch();
   const trackEvent = useTrackEvent();
+  const pushToHistory = usePushToHistory();
+  const intl = useIntl();
 
   React.useEffect(() => {
     dispatch(changeRoute(DASHBOARD_ROUTE.key));
     trackEvent('Home - Visit Dashboard Page');
   }, []);
 
+  const newsBannerProps = {
```

Comment:
We should send this all into its own component, so we only need to change that once we change the banner